elife/config/etc-php-7.0-fpm-php.ini

```diff
@@ -825,7 +825,7 @@ allow_url_include = Off
 
 ; Default timeout for socket based streams (seconds)
 ; http://php.net/default-socket-timeout
-default_socket_timeout = 60
+default_socket_timeout = 5
```

Comment:
Long timeouts should be set explicitly.

Reply:
Do we have any idea of where this is used? I think curl, being its own extensions, may not be influenced by this?

Reply:
Not cURL, but `file_get_contents()` etc.